### DISCUSSION TITLE
Twice-a-month Recurrence

### DISCUSSION THREAD

**Discussion by webtroter** [firefly-iii/firefly-iii#8147] [Firefly III]
Hello!

I'm being paid twice a month.

I figure I can do two monthly recurrence for each payday in the month. But if there could be a nicer way it would be nice.

Alternatively, having the option to duplicate  Recurring transactions to help with the twice a month recurrence.

At first I search for bi-monthly, and I got result where the author suggest using `skip` to 1, but that for every other month. I basically need to set skip at 0.5 😅

Thank you

**Comment by JC5** [comment]
You can do every two weeks but that's it im afraid

**Comment by webtroter** [comment]
Well, that's what I was thinking.

Bi-weekly doesn't work because that 26 times a year vs 24 for twice a month.

Thank you for the reply though.

Please consider my feature request for easy duplication of recurrences

Thank you again

W.
